### PR TITLE
Extend gitignore

### PR DESCRIPTION
**Pull request by martonmiklos**
Adding more generated/unrelated files to the .gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,7 +25,42 @@ imgui.ini
...
+CPackConfig.cmake
+CPackSourceConfig.cmake
+
+include/version.h
+asset/openboardview.appdata.xml
+asset/openboardview.desktop
+share/appdata/openboardview.appdata.xml
+share/applications/openboardview.desktop
+share/icons/hicolor/scalable/apps/openboardview.svg
+share/mime/packages/openboardview.xml
```
Comment (MattSturgeon) [inline .gitignore]: Is it worth just ignoring /asset and `/share`?

Comment (martonmiklos) [reply .gitignore]: The asset folder contains files in git:
https://github.com/OpenBoardView/OpenBoardView/tree/master/asset

But with the share I agree with you. Give me a minute to fix it.

Comment (MattSturgeon) [reply .gitignore]: If we already have the directory I'd probably prefer having a .gitignore file _in_ the asset dir

Comment (martonmiklos) [reply .gitignore]: Makes sense, fixed it!

**Pull request merged**